lib/shelley/src/Cardano/Wallet/Shelley/MinimumUTxO.hs

```diff
@@ -174,3 +203,17 @@ unsafeCoinFromCardanoApiCalculateMinimumUTxOResult = \case
             , "unexpected error:"
             , show e
             ]
+
+-- | Uses the ledger to compute a minimum ada quantity for the Babbage era.
+--
+computeLedgerMinimumCoinForBabbage
+    :: Babbage.PParams StandardBabbage
+    -> Address
+    -> TokenBundle
+    -> Coin
+computeLedgerMinimumCoinForBabbage pp addr tokenBundle =
+    Coin $ fromIntegral @Integer @Natural result
```

Comment:
This silently wraps negative values unlike [unsafeLovelaceToWalletCoin](https://github.com/input-output-hk/cardano-wallet/blob/c1f17e791ed9f69b752ef4359be1a1a940c5a3e7/lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs#L2146-L2155) for other eras. At least for consistency I think we should do it here as well.

Reply:
> This silently wraps negative values unlike [unsafeLovelaceToWalletCoin](https://github.com/input-output-hk/cardano-wallet/blob/c1f17e791ed9f69b752ef4359be1a1a940c5a3e7/lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs#L2146-L2155) for other eras. At least for consistency I think we should do it here as well.

Excellent point. I've fixed this in 529c7dee4b079b9333c319b14c91c69cab065376.